Include/pyport.h

```diff
@@ -713,13 +713,21 @@ extern char * _getpty(int *, int, mode_t, int);
 #endif
 
 #if LONG_BIT != 8 * SIZEOF_LONG
+#if !defined(_AIX)
```

Comment:
IMO, it would be better to use `#ifndef`.

Reply:
Thanks for the review.
I used #if plus comments in the hope that was more legible, Your comment indicates this did not work, so switched to #ifndef and removed additional macro comments.